bootstrap/postgres/configs.go

```diff
@@ -483,10 +507,11 @@ func (cr configRepository) retrieveAll(owner string, filter bootstrap.Filter) (s
 	return fmt.Sprintf(template, f), params
 }
 
-func (cr configRepository) rollback(content string, tx *sqlx.Tx) {
+func (cr configRepository) rollback(content string, tx *sqlx.Tx) error {
 	if err := tx.Rollback(); err != nil {
-		cr.log.Error(fmt.Sprintf("Failed to rollback due to %s", err))
+		return fmt.Errorf("%s: %w", content, err)
 	}
+	return nil
 }
```

Comment:
Please use errors package to wrap errors 

And we need to pass  the content as new error 


```suggestion
func (cr configRepository) rollback(content string, tx *sqlx.Tx) error {
    errMsg := errors.New(content)
	if err := tx.Rollback(); err != nil {
        errRollback := errors.New("failed to rollback")
        return errors.Wrap(errMsg, errors.Wrap(errRollback, err))
	}
	return errMsg
}
```